opcache-gui/provision.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Checkout Opcache GUI to provide a dashboard for viewing statistics
+
+if [[ ! -d "/srv/www/default/opcache-gui" ]]; then
+	echo -e " * Downloading Opcache GUI, see https://github.com/amnuts/opcache-gui"
+	mkdir -p /srv/www/default/opcache-gui
+	cd /srv/www/default/opcache-gui
+else
+	echo -e " * Updating Opcache GUI"
+	cd /srv/www/default/opcache-gui
```

Comment:
Move this line out of the if structures as it's duplicated and happens on both branches